README.md

```diff
@@ -29,6 +29,12 @@ Since a database failover is usually identified by reaching a network or a conne
 
 Enhanced Failure Monitoring (EFM) is a feature available from the [Host Monitoring Connection Plugin](./docs/using-the-jdbc-driver/using-plugins/UsingTheHostMonitoringPlugin.md#enhanced-failure-monitoring) that periodically checks the connected database node's health and availability. If a database node is determined to be unhealthy, the connection is aborted (and potentially routed to another healthy node in the cluster).
 
+### Using the AWS JDBC Driver with RDS Multi-AZ DB Clusters
+The AWS JDBC Driver is also optimized for fast switchover on [AWS RDS Multi-AZ DB Clusters](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/multi-az-db-clusters-concepts.html).
```

Comment:
I don't think we've used the term switchover before.

I think this paragraph should mention:
- what are those new clusters and ensure that the driver now fully supports them